<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QDialog" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>690</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>471</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="hole_12">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.345549, y1:0.130682, x2:0.670157, y2:1, stop:0 rgba(255, 178, 102, 255), stop:0.78534 rgba(235, 148, 61, 255), stop:1 rgba(89, 48, 18, 255));
color: black;
border-radius: 8px;
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="hole_11">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.345549, y1:0.130682, x2:0.670157, y2:1, stop:0 rgba(255, 178, 102, 255), stop:0.78534 rgba(235, 148, 61, 255), stop:1 rgba(89, 48, 18, 255));
color: black;
border-radius: 8px;
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="hole_10">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.345549, y1:0.130682, x2:0.670157, y2:1, stop:0 rgba(255, 178, 102, 255), stop:0.78534 rgba(235, 148, 61, 255), stop:1 rgba(89, 48, 18, 255));
color: black;
border-radius: 8px;
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="hole_9">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.345549, y1:0.130682, x2:0.670157, y2:1, stop:0 rgba(255, 178, 102, 255), stop:0.78534 rgba(235, 148, 61, 255), stop:1 rgba(89, 48, 18, 255));
color: black;
border-radius: 8px;
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="hole_8">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.345549, y1:0.130682, x2:0.670157, y2:1, stop:0 rgba(255, 178, 102, 255), stop:0.78534 rgba(235, 148, 61, 255), stop:1 rgba(89, 48, 18, 255));
color: black;
border-radius: 8px;
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="hole_7">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.345549, y1:0.130682, x2:0.670157, y2:1, stop:0 rgba(255, 178, 102, 255), stop:0.78534 rgba(235, 148, 61, 255), stop:1 rgba(89, 48, 18, 255));
color: black;
border-radius: 8px;
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_1">
      <item>
       <widget class="QPushButton" name="hole_1">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.345549, y1:0.130682, x2:0.670157, y2:1, stop:0 rgba(255, 178, 102, 255), stop:0.78534 rgba(235, 148, 61, 255), stop:1 rgba(89, 48, 18, 255));
color: black;
border-radius: 8px;
</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="hole_2">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.345549, y1:0.130682, x2:0.670157, y2:1, stop:0 rgba(255, 178, 102, 255), stop:0.78534 rgba(235, 148, 61, 255), stop:1 rgba(89, 48, 18, 255));
color: black;
border-radius: 8px;
</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="hole_3">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.345549, y1:0.130682, x2:0.670157, y2:1, stop:0 rgba(255, 178, 102, 255), stop:0.78534 rgba(235, 148, 61, 255), stop:1 rgba(89, 48, 18, 255));
color: black;
border-radius: 8px;
</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="hole_4">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.345549, y1:0.130682, x2:0.670157, y2:1, stop:0 rgba(255, 178, 102, 255), stop:0.78534 rgba(235, 148, 61, 255), stop:1 rgba(89, 48, 18, 255));
color: black;
border-radius: 8px;
</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="hole_5">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.345549, y1:0.130682, x2:0.670157, y2:1, stop:0 rgba(255, 178, 102, 255), stop:0.78534 rgba(235, 148, 61, 255), stop:1 rgba(89, 48, 18, 255));
color: black;
border-radius: 8px;
</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="hole_6">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.345549, y1:0.130682, x2:0.670157, y2:1, stop:0 rgba(255, 178, 102, 255), stop:0.78534 rgba(235, 148, 61, 255), stop:1 rgba(89, 48, 18, 255));
color: black;
border-radius: 8px;
</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="kalah_1">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>100</y>
     <width>70</width>
     <height>180</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.345549, y1:0.130682, x2:0.528796, y2:1, stop:0 rgba(255, 178, 102, 255), stop:0.78534 rgba(235, 148, 61, 255), stop:1 rgba(89, 48, 18, 255));
color: black;
border-radius: 8px;
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="kalah_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>70</width>
     <height>180</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.345549, y1:0.130682, x2:0.528796, y2:1, stop:0 rgba(255, 178, 102, 255), stop:0.78534 rgba(235, 148, 61, 255), stop:1 rgba(89, 48, 18, 255));
color: black;
border-radius: 8px;
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.172775, y1:0, x2:0.73822, y2:0.926, stop:0 rgba(129, 0, 0, 255), stop:1 rgba(0, 0, 144, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="whose_turn_label">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>20</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>35</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgba(255, 255, 255, 51);
border-radius: 8px;</string>
   </property>
   <property name="text">
    <string>AI turn</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>kalah_1</zorder>
  <zorder>kalah_2</zorder>
  <zorder>whose_turn_label</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
